cmd/medius/common/registry.go

```diff
@@ -87,14 +87,14 @@ var staticRegistry = []Entry{
 	},
 	{
 		Artifacts: []api.Artifact{
-			debian.New("11", "bullseye", "x86_64", &docs.UserData{Username: "debian"}, nil),
-			debian.New("11", "bullseye", "aarch64", &docs.UserData{Username: "debian"}, nil),
+			debian.New("11", "bullseye", "x86_64", &docs.UserData{Username: "debian"}, defaultEnvVariables("u1.nano", "debian")),
```

Comment:
Nano is very small, maybe stick with medium instead like the others?

Reply:
IIUC, `nano` means: 1 CPU and 512MiB. Debian recommends 1 CPU and 512MB. So yes, nano is not enough for this (I got tricked by unit conversions :upside_down_face:)
Medium means 1 CPU and 2GiB, which it is above what Debian needs. What you think about `micro`?

Reply:
It's the bare minimum, but other disks recommend `medium` too. Makes sense to me to stick with it?

Reply:
All right, let's stick with `medium`.